src/email-assets/email-templates/insights-automation-services.html

```diff
@@ -0,0 +1,372 @@
+    }
+
+    .rh-content__block-title {
+      font-weight: bold;
+    }
+
+    .rh-footer__text {
+      font-size: 14px;
+    }
+
+    .rh-footer__text a {
+      color: #0066cc;
+    }
+
+    .rh-content__spacer {
+      font-size: 0;
+      line-height: 0;
+      height: 20px;
+      background: #ededed;
+    }
+
+    .rh-cta-link {
+/*       width: 100%;
+      height: 100%; */
+      display:block;
+      padding: 10px 20px;
+      background-color: #0066cc;
+      border-radius: 3px;
+      text-decoration:none;
+      font-weight:bold;
+    }
+
+    .rh-cta-link a {
+      color:#ffffff;
+      text-decoration: none;
+    }
+
+    .rh-m-block {
+      display: block;
+    }
+
+    .rh-data-table {
+      width: 100%;
+    }
+
+    .rh-data-table thead th {
+      font-weight: 700;
+      text-align: left;
+    }
+
+    .rh-data-table tbody th {
+      font-weight: 500;
+      text-align: left;
+      width: 200px;
+    }
+
+    .rh-data-table.rh-m-bordered th {
+      padding: 10px;
+      text-align: left;
+      font-size: 14px;
+      font-weight: 500;
+      text-transform: uppercase;
+      background-color: #bee1f4;
+    }
+
+    .rh-data-table.rh-m-bordered td {
+      padding: 10px;
+      text-align: left;
+    }
+
+    .rh-data-table.rh-m-bordered {
+      border-collapse: collapse;
+    }
+
+    .rh-data-table caption {
+      text-align: left;
+      padding-bottom: 10px;
+    }
+
+    .rh-m-bordered th,
+    .rh-m-bordered td {
+      border-width: 1px;
+      border-color: #8A8D90;
+      border-style: solid;
+    }
+
+    .rh-color__red {
+      color: #a30000;
+    }
+
+    .rh-color__orange {
+      color: #795600;
+    }
+
+    .rh-color__green {
+      color: #486b00;
+    }
+
+  </style>
+
+  </head>
+
+  <!-- text-size-adjust: none; -webkit-text-size-adjust: none; -moz-text-size-adjust: none; -ms-text-size-adjust: none; -->
+  <body width="100%" class="rh-body">
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tbody>
+        <tr>
+          <td class="rh-page">
+            <!--Table within the larger table; contains email content (600px)-->
+            <table role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="600">
+              <tbody>
+                <tr>
+                  <td class="rh-head">
+                    <table class="rh-masthead" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="600">
+                      <tbody>
+                        <tr>
+                          <!--Red Hat logo-->
+                          <td align="center" class="rh-masthead__brand">
+                            <a href="#" target="_blank" class="rh-masthead__brand-link">
+                              <img src="https://raw.githubusercontent.com/RedHatInsights/frontend-assets/master/src/email-assets/logo_insights.png" alt="Red Hat logo" width="152" height="60" class="rh-masthead__brand-img" />
```

Comment:
use c.rh.c

Reply:
There are a few places for this, I definitely missed some, but if we can swap all of these out, that'd be great